src/decision/include/GpsManager.h

```diff
@@ -1,19 +1,20 @@
 /*
  * Created By: Gareth Ellis
- * Created On: October 20, 2016
+ * Created On: May 17, 2017
  * Description: Manages the GPS waypoints we are given
  *              (publishing the next one once we've arrived
  *              at the current one)
  */
```

Comment:
Wow. The diff on this file looks *TERRIBLE*. What happened here is I basically re-wrote it from scratch, so I suggest you just review the file by itself as if I were submitting it for the first time.